### DISCUSSION TITLE
Any documents explaining the scheme of running the scripts?

### DISCUSSION THREAD

**Discussion by shqwu** [aquacropos/aquacrop#59] [General]
I really appreciate for providing the python-version AquaCrop! I am trying to apply machine learning to the AquaCrop so I need to have my own codes but scripts from this package helps me a lot!!!! Just wondering if there is some documentation that summarizes the linkages and applications of all these scripts? Thanks!

**Comment by thomasdkelly** [comment]
Hi the docs have just been updated and the tutorial notebooks should have all the info you need as well. If you want to know anything else let me know

Tom

**Reply by shqwu** [reply]
Awesome! Thank you so much! :)
